pyproject.toml

```diff
@@ -39,13 +39,16 @@ typer-config = "^1.4.0"
 phonopy = "^2.23.1"
 seekpath = "^1.9.7"
 spglib = "^2.3.0"
-torch-dftd = "^0.4.0"
+torch-dftd = "0.4.0"
```

Comment:
Are we also certain we want to unpin torch? Presumably it was pinned for a reason.

Reply:
My brain is working backwards. This is now *strictly* pinned to `0.4.0`. (As was numpy being strictly pinned) for some reason I thought this was specifying minimum version now

Reply:
For reference, I think this is only used by [mace](https://github.com/ACEsuit/mace/blob/main/mace/calculators/foundations_models.py). Pinning, as it now is, is probably safest.

Reply:
yes correct... more precisely mace_mp for mace naked we need a sum calculator from ase... probably similar for others...